src/service/crypto-impl.js

```diff
@@ -30,11 +30,22 @@ export class Crypto {
     /** @private {!Window} */
     this.win_ = win;
 
+    let subtle = null;
+    let isLegacyWebkit = false;
+    if (win.crypto) {
+      if (win.crypto.subtle) {
+        subtle = win.crypto.subtle;
+      } else if (win.crypto.webkitSubtle) {
+        subtle = win.crypto.webkitSubtle;
+        isLegacyWebkit = true;
+      }
+    }
+
     /** @private @const {?webCrypto.SubtleCrypto} */
-    this.subtle_ = getSubtle(win);
+    this.subtle_ = subtle;
```

Comment:
Nit: Any reason to not just use this.subtle_ on lines 37, 39?

Reply:
Then it couldn't be `@const` anymore.